src/chat/components/cards/NotesCard.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useIntl } from 'react-intl';
+import {
+  Button,
+  Card,
+  CardActions,
+  Collapse,
+  Divider,
+} from '@material-ui/core';
+import CardHeaderComp from './CardHeader';
+import NotesInput from './NotesInput';
+import NotesList from './NotesList';
+import { useSelector } from 'react-redux';
+import { currentUserSelector, loadingCurrentUserSelector } from '../../redux';
+import { CardDetailSkeleton } from '../LoadingScreens';
+
+const NOTES = [
```

Comment:
This is only placeholder data until we have the backend endpoint, right?

Reply:
That's correct :)